### PR TITLE
Add default xp drop recoloring

### LINKED ISSUE #5242: Recolour Official Experience Drops
**Is your feature request related to a problem? Please describe.**
The official experience drop colour options are both ugly and do not match the overall exp counter. 
Osbuddy gives the option to recolour both the drops and the overall counter, I would love to see this in RuneLite!

**Describe the solution you'd like**
Similar to the chat recolour plugin, a option to recolour:
The official experience drops
The official experience counter

**Additional context**
Osbuddy:
screen shot 2018-07-18 at 8 59 52 pm
Official:
screen shot 2018-07-18 at 9 00 25 pm

### PR DESCRIPTION
**Pull request by astaninger**
Closes #5242

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/experiencedrop/XpDropConfig.java [hunk 1]
```diff
@@ -75,4 +75,15 @@ default Color getMagePrayerColor()
 	{
 		return new Color(0x15, 0x80, 0xAD);
 	}
+
+	@ConfigItem(
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/experiencedrop/XpDropConfig.java]: You probably should make this null by default and respect default vanilla settings by default

File: runelite-client/src/main/java/net/runelite/client/plugins/experiencedrop/XpDropConfig.java [hunk 2]
```diff
@@ -71,8 +71,13 @@ default Color getRangePrayerColor()
...
-	}
+	default Color getMagePrayerColor() { return new Color(0x15, 0x80, 0xAD); }
+
+	@ConfigItem(
+			keyName = "default Color",
+			name = "Default Color",
+			description = "XP drop color when combat prayer is inactive",
+			position = 4
+	)
+	default Color getDefaultColor() { return null; }
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/experiencedrop/XpDropConfig.java]: Check chatColors config I believe we just dont add method body for no value colors

File: runelite-client/src/main/java/net/runelite/client/plugins/experiencedrop/XpDropPlugin.java [hunk 3]
```diff
@@ -124,7 +124,7 @@ else if (!widget.getText().isEmpty())
 		PrayerType prayer = getActivePrayerType();
 		if (prayer == null)
 		{
-			resetTextColor(widget);
+			widget.setTextColor(config.getDefaultColor().getRGB());
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/experiencedrop/XpDropPlugin.java]: Null check?

**Comment by TzZek** [comment]
Sort of what my #3028 did

File: runelite-client/src/main/java/net/runelite/client/plugins/experiencedrop/XpDropPlugin.java (hunk 3 above)
Comment (Adam-) [reply runelite-client/src/main/java/net/runelite/client/plugins/experiencedrop/XpDropPlugin.java]: resetTextColor becomes unused here? and I think it is needed in order to reset changed colors due to the prayer flicking.

**Comment by montrers** [comment]
bump?

**Comment by Adam-** [comment]
added in #13729

**Pull request closed without merging**